great_expectations/datasource/fluent/interfaces.py

```diff
@@ -448,10 +440,11 @@ def _parse_order_by_sorters(
     ) -> List[Sorter]:
         return Datasource.parse_order_by_sorters(order_by=order_by)
 
-    def sort_batches(self, batch_list: List[Batch], partitioner: PartitionerProtocol) -> None:
+    def sort_batches(self, batch_list: List[Batch], partitioner: PartitionerI) -> None:
         """Sorts batch_list in place in the order configured in this DataAsset.
         Args:
             batch_list: The list of batches to sort in place.
+            partitioner:
```

Comment:
Please add a description for partitioner.

Reply:
updated in [ca76ec2](https://github.com/great-expectations/great_expectations/pull/9792/commits/ca76ec27b37b81148fbfe84860311f30b49716fe)